### ISSUE TITLE
Transferred pokemon are not deleted

### ISSUE THREAD

**Issue by qwer4790** [justinleewells/pogo-optimizer#185]
Here is how this bug happen:
1) Use pogo-optimizer to check IVs as usual.
2) Disconnect from proxy
3) Re-login pokemon go and transfer some pokemons within normal network
4) Back to pogo-optimizer, previously transferred pokemon still on the page

Is there any command to clear the cache or something?

**Comment by justinleewells** [comment]
You just need to update the app.

**Comment by macmanchomp** [comment]
As clarification, when running the latest version of pogo-optimizer if something like this happens at any point you can click the logout button and relaunch the app from your phone to get a fresh view, but this should never happen. The page should simply update with the current list and dump all of the old stuff. So in short, stop the proxy (ctrl+c) and run
```
git fetch origin master
git pull
npm install
npm start
```
This should check your version against the latest, update what needs updating, install the new dependencies, then start the proxy back up. You'll likely need to download the ca.crt or ca.pem again after doing this. 

Note: This is assuming you are running this from a bash command line on a *nix based system. Docker is completely alien to me, maybe someone else can chime in on the update process for that if you need it.

**Comment by qwer4790** [comment]
Yes I use docker, so it might have some issue..I will check it again after update.

**Comment by dtandersen** [comment]
I've noticed that when a transfer is missed it's often updated after the next transfer.
